be/src/exec/pipeline/olap_table_sink_operator.h

```diff
@@ -0,0 +1,80 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <utility>
+
+#include "exec/pipeline/fragment_context.h"
+#include "exec/pipeline/operator.h"
+#include "exec/tablet_sink.h"
```

Comment:
could you include this in cpp files to make header including as few as possible.